library/cc/engine_builder.h

```diff
@@ -33,6 +33,7 @@ class EngineBuilder {
   EngineBuilder& setAppVersion(const std::string& app_version);
   EngineBuilder& setAppId(const std::string& app_id);
   EngineBuilder& setDeviceOs(const std::string& app_id);
+  EngineBuilder& setDecompressor(bool decompressor_on);
```

Comment:
I could imagine int he long term this being a yaml config string but I'd like to keep it simple for now :-)

Reply:
This is fine for now. I'd actually like to move away from passing in YAML strings into the builder altogether (and have 
a general alternative approach in mind).